load_and_plot_dm.py

```diff
@@ -0,0 +1,47 @@
+from petsc4py import PETSc
```

Comment:
Should these python scripts go in some `scripts` directory? Also these helper programs - should they go in say a `tools` directory rather than the root? Maybe bother go in a `tools` directory?